rajawali/src/androidTest/java/org/rajawali3d/scenegraph/OctreeTest.java

```diff
@@ -0,0 +1,47 @@
+package org.rajawali3d;
+
+import static org.junit.Assert.*;
+import org.junit.*;
+
+import org.rajawali3d.scenegraph.Octree;
+import org.rajawali3d.math.vector.Vector3;
+
+/**
+ * @author Jared Woolston (jwoolston@keywcorp.com)
```

Comment:
Do we have two people named Jared now? :laughing: 

You can remove the comment entirely or put your info. Either way.